include/eve/algo/max_element.hpp

```diff
@@ -0,0 +1,95 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/algo/common_forceinline_lambdas.hpp>
+#include <eve/algo/find_last.hpp>
+#include <eve/algo/max_value.hpp>
+
+namespace eve::algo
+{
+template<typename TraitsSupport> struct max_element_ : TraitsSupport
+{
+  template<relaxed_range Rng, typename Less>
+  EVE_FORCEINLINE auto operator()(Rng&& rng, Less less) const -> unaligned_iterator_t<Rng>
+  {
+    if( rng.begin() == rng.end() ) return unalign(rng.begin());
+    auto v = *max_value[TraitsSupport::get_traits()](EVE_FWD(rng), less);
+    // less(x, max) -> false only for max
+    return eve::algo::find_last_if_not[TraitsSupport::get_traits()](EVE_FWD(rng), bind_second{less, v});
```

Comment:
I assume we take find_last to find the latest max in order to be stepanovés compliant ?

Reply:
Yes! As it says in very pretty docs :)